index.ts

```diff
@@ -21,34 +22,34 @@ export function setCredentials(credentials: any) {
     musicStore.setCredentials(credentials);
 }
 
-/**
+/** review name
  * Get the accessToken provided via through the setCredentials api
- * @returns {string} the access token string
+ * @returns {string } the access token string
  */
 export function getAccessToken() {
     return musicStore.credentialByKey("accessToken");
 }
 
-/**
+/** review name
  * Checks if the Spotify desktop is running or not
- * @returns {Promise<boolean>}
+ * @returns {Promise<boolean> }
  */
 export function isSpotifyRunning() {
     return isRunning(PlayerName.SpotifyDesktop);
 }
 
-/**
+/** review name
  * Checks if the iTunes desktop is running or not
- * @returns {Promise<boolean>}
+ * @returns {Promise<boolean> }
  */
 export function isItunesRunning() {
     return isRunning(PlayerName.ItunesDesktop);
 }
 
-/**
+/** review name
  * Checks if one of the specified players is running
- * @param player {spotify|spotify-web|itunes}
- * @returns {Promise<boolean>}
+ * @param player {spotify|spotify-web|itunes }
+ * @returns {Promise<boolean> }
  */
 export async function isRunning(player: PlayerName) {
```

Comment:
"isPlayerRunning" (less ambiguous).

Also, in the description, consider "Checks if a player is running"